src/main/java/com/boot/user/exception/GlobalExceptionHandler.java

```diff
@@ -92,4 +92,12 @@ public ResponseEntity<ApiError> methodArgumentNotValidException(
 				methodArgumentNotValidException);
 	}
 
+	@ExceptionHandler(EmailAlreadyUsedException.class)
+	@ResponseStatus(HttpStatus.BAD_REQUEST)
+	public ResponseEntity<ApiError> emailAlreadyUsedException(
+			EmailAlreadyUsedException emailAlreadyUsedException)
+	{
+		return createResponseEntity(HttpStatus.BAD_REQUEST,
```

Comment:
with this message structure won't be easy to work on the FE but if you want so, we can keep it so